test-schemas/ajv.js

```diff
@@ -0,0 +1,11 @@
+const Ajv = require('ajv');
+ajv = new Ajv({
```

Comment:
Can we use `var` here? Otherwise we're setting a global?

Reply:
Or `const`, even, since we're using Node 10?

Reply:
Yes missed it